pkg/broker/binding_operations.go

```diff
@@ -155,6 +155,8 @@ func (b Broker) Unbind(ctx context.Context, instanceID string, bindingID string,
 
 	client, p, err := b.getClient(ctx, instanceID, details.PlanID, nil)
 	if err != nil {
+		logger.Errorw("Failed to get existing client", "error", err)
```

Comment:
Could you add some more logging to the code where the parsing problem was happening? (this is to make debugging easier for us the next time we have a data consistency problem)

Reply:
`getClient()` returns multiple kinds of errors so it is pretty verbose already.
Adding the same line for `Bind` operation.